### PR TITLE
apply option key symbol conditionally. change command key symbol from…

### PR DESCRIPTION
**Pull request by Lisiadito**
… ^ to ctrl. adjust css

| Q                | A
| ---------------- | ---
| Bug fix?         | no
| New feature?     | no
| BC breaks?       | no
| Deprecations?    | no
| New tests added? | not needed
| Fixed tickets    | #1635 
| License          | MIT

### Description
Wrong shortcut keys where shown on Linux and Windows. 
--

### CODE REVIEW

**Comment by Jocs** [comment]
@Lisiadito Thanks for you PR, please make sure the shortcuts align right.

截屏2019-11-07下午9 39 53

**Comment by Lisiadito** [comment]
Sure will fix that

File: src/muya/lib/ui/quickInsert/index.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .ag-quick-insert {
-  width: 324px;
+  width: auto;
```
Comment (Jocs) [inline src/muya/lib/ui/quickInsert/index.css]: Why set it to `auto`? and the quick insert panel mess up.

截屏2019-11-07下午11 57 09


Set it to `350px`, and works.

Comment (Lisiadito) [reply src/muya/lib/ui/quickInsert/index.css]: mhh strange looked fine when I checked it. I'll undo that..

Comment (Jocs) [reply src/muya/lib/ui/quickInsert/index.css]: OK.

**Comment by Lisiadito** [comment]
sorry for the mess with the commits. I guess you can squash them when merging it.

**Comment by Jocs** [comment]
@Lisiadito Thanks again, and it looks much better than before.

**Comment by Jocs** [comment]
>sorry for the mess with the commits. I guess you can squash them when merging it.

Never mind, we'll squash merge it.

**Pull request merged**